webpack.config.js

```diff
@@ -93,7 +93,7 @@ module.exports = (env = {}) => {
                       modules: false,
                       shippedProposals: true,
                       useBuiltIns: "usage",
-                      targets: { browsers: ["last 2 versions"] },
+                      targets: { browsers: ["last 2 versions"], chrome: 41 },
```

Comment:
I think the format is more like this:

```js
targets: {
  browsers: [
    "Chrome >= 41",
    "last 2 versions",
  ],
}
```

Can you run `rm -rf tmp/ && npm run build` with `debug: true` for `@babel/preset-env` to see the effect? `@babel/preset-env` should log out, which chrome version it now targets.

Reply:
It log out version 41 of chrome in targets. I checked also which version of chrome the google search spiders currently uses and it's 41.